.circleci/config.yml

```diff
@@ -34,3 +109,47 @@ jobs:
       - store_artifacts:
           path: vc4cl-stdlib-0.4-Linux.deb
           distination: vc4cl-stdlib-0.4-Linux.deb
+      - persist_to_workspace:
+          root: /root/project
+          paths:
+            - build/libVC4CC.so.0.4
+            - build/test/TestVC4C
+            - build/cpptest-lite/src/cpptest-lite-project-build/libcpptest-lite.so.0.9
+            - testing
+            - example
+            - vc4cl-stdlib-0.4-Linux.deb
+  test:
+    docker:
+      - image: nomaddo/native
+    steps:
+      - attach_workspace:
+          at: /root/project
+      - run: dpkg -i vc4cl-stdlib-0.4-Linux.deb
+      - run: ln -s `pwd`/build/libVC4CC.so.0.4 /usr/lib/libVC4CC.so.1.2
+      - run: ln -s `pwd`/build/cpptest-lite/src/cpptest-lite-project-build/libcpptest-lite.so.0.9 /usr/lib/libcpptest-lite.so.1.1.2
+      - run: ldconfig
+      - run: build/test/TestVC4C
```

Comment:
TestVC4C runs about 15 mins (on my desktop PC), so running it twice on every build is not an option.
I selected a few test-cases which run quite fast (and am working on adding some more):

    build/test/TestVC4C --output=plain --mode=verbose --test-instructions --test-operators

Same for the test beneath